src/ai/backend/client/cli/session/execute.py

```diff
@@ -227,6 +227,8 @@ def format_stats(stats):
 def prepare_resource_arg(resources):
     if resources:
         resources = {k: v for k, v in map(lambda s: s.split("=", 1), resources)}
+        if resources["mem"] and "m" not in resources["mem"] and "g" not in resources["mem"]:
+            resources["mem"] += "m"
```

Comment:
Units of resources should be handled very strictly and commonly among components because many components shares such resource options and handling resources is critical to our system.

Also, if we also consider cases like shared memory (shmem), we need more general unit handling.
We defined [`BinarySize` type in common](https://github.com/lablup/backend.ai/blob/c0bbaa9012fa3120fa3c657bed963fd96835bc76/src/ai/backend/common/types.py#L434-L577). How about using it??

Plus, I think we can add type hints to this function!

Reply:
Thank you for feedback. I will apply them to this CL!